rulebook.tex

```diff
@@ -612,26 +670,182 @@ \subsection{Machines}
     \end{minipage}
 }
 \vspace{-1ex}
-\caption{The different MPS stations}
+\caption{The different \ac{MPS} stations}
 \end{figure}
 
 \begin{table}[!tb]
     \centering
     \mytable{
         \begin{tabularx}{\linewidth}{l|l|X}
             Type & Distribution & (Final) processing time[s]\\\hline
-            Base Station (BS) & 1 per team & minimum physical time\\
-            Cap Station (CS) & 2 per team & $t_2 = \SItextrange{15:25}{\sec}$\\
-            Cap Station (CS) & 2 per team & $t_2 = \SItextrange{15:25}{\sec}$\\
-            Ring Station (RS) & 2 per team &$t_3 = \SItextrange{40:60}{\sec}$\\
-            Storage Station (SS) & 1 per team & minimum physical time\\
-            Delivery Station (DS) & 1 per team &$t_5 =\SItextrange{20:40}{\sec}$
+            \acf{BS} & 1 per team & minimum physical time\\
+            \acf{CS} & 2 per team & $t_2 = \SItextrange{15:25}{\sec}$\\
+            \acf{RS} & 2 per team &$t_3 = \SItextrange{40:60}{\sec}$\\
+            \acf{SS} & 1 per team & minimum physical time\\
+            \acf{DS} & 1 per team &$t_5 =\SItextrange{20:40}{\sec}$
         \end{tabularx}
     }
-    \caption{MPS type, distribution and processing times}
+    \caption{\ac{MPS} type, distribution and processing times}
     \label{tab:processing-times}
 \end{table}
 
+\subsubsection{Physical Description of Machines}
+
+The stations have a rectangular base shape of \SI{0,35 x 0,7}{\metre}% chktex 29
+with a height of about \SI{1}{\metre} depending on machine type.
+A machine is movable by four wheels with \SI{0.1}{\metre} clearance.
+Both narrow sides of the trolley are closed by plexiglas and have a handle.
+All physical interfaces like conveyor belt inputs and outputs, shelves, and
+slides for additional bases on ring stations are accessible at
+\SI{89.8}{\centi\metre} height\footnote{Note however, that due to small
+  variances and unevenness in the floor there may likewise be small variances
+  in the working height of some or all parts of a station.}.
+Working space between guiding lanes is \SI{4.5}{\centi\metre}. Setup lanes and
+shelves feature approximately the same space for handling and
+adjusting. All diffuse sensors (input side) have been removed to allow
+for infrared-emitting cameras.  To simplify the delivery on the belt,
+each machine will be equipped with a narrowing cone on its input side
+(\reffig{fig:narrow-cone}).  You can order them from Festo Didactic SE
+(C. Deppe) or download the STL model file from the \ac{RCLL} site.
```

Comment:
I think a reference to the download would be nice. Otherwise it might be frustrating googleing :D

Reply:
good point, i actually cannot find the stl. So is there even a link we can point to?